tests/tests_api.py

```diff
@@ -733,15 +733,6 @@ def setUpClass(cls):
             url=serverurl, verbose=clverb, show_curl=showcurl
         )
 
-        print(
```

Comment:
@pothiers this is repeated from the setUpClass under the `class SparclClientTest`, so I wasn't sure if it needed to be included again in the setUpClass for `class AuthTest`? If yes, then I'll probably add something like "Running Auth Client Tests" at the beginning of the print statement.

Reply:
I didn't catch that it was repeated. The goal should be to see the context exactly once in any test.  That might not be possible since the command that runs the test might specify just one test class to run.  If it is possible to show it just once (maybe there is a setup function that runs before ANY tests?) then best to remove them from everywhere except for the single place.  Otherwise, the context print will have to be in multiple places but could probably use a global variable to conditionally print and make it never print multiple times.